tests/test_community_scripts_proxmoxve.py

```
@@ -0,0 +1,296 @@
+    def _patch_https_connection(self, payload: object, status: int = 200) -> mock._patch:
+        return mock.patch.object(
+            self.module.http.client,
+            "HTTPSConnection",
+            return_value=FakeHTTPSConnection(payload, status=status),
+        )
+
+    def test_fetch_scripts_rejects_non_list_payload(self) -> None:
+        with self._patch_https_connection({"scripts": []}):
+            scripts = self.module._fetch_scripts()
+
+        self.assertEqual(scripts, [])
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)